1.3.c

```diff
@@ -0,0 +1,121 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+#define MAXIMUMSIZE 256

```

Comment:
```suggestion
#define MAXIMUM_SIZE 256
```
Слова в случае define констант разделяются этим символом (-0.1)